### ISSUE TITLE
NQ training qrels missing

### ISSUE THREAD

**Issue by cadurosar** [beir-cellar/beir#108]
Hey, I was looking into the training sets contained in BEIR, and it seems that the NQ one is missing. The the one provided in the current link only has the test set (https://public.ukp.informatik.tu-darmstadt.de/thakur/BEIR/datasets/nq.zip). Could you help me with this?

**Comment by thakur-nandan** [comment]
Hi @cadurosar, 

The `nq` folder provided only contains the test set. For nq training qrels you can have a look into [https://public.ukp.informatik.tu-darmstadt.de/thakur/BEIR/datasets/nq-train.zip](https://public.ukp.informatik.tu-darmstadt.de/thakur/BEIR/datasets/nq-train.zip). T

The reason for this is because the corpus used for both training and test is different. Training set contains passage duplicates which are suitable for training, however not for validation.

**Comment by cadurosar** [comment]
Thanks @thakur-nandan!
